### PR TITLE
Tools: added name to GIT_Success.txt

### PR DESCRIPTION
**Pull request by cavesLezet**
this pr adds my name to GIT_Success.txt)

### CODE REVIEW

File: Tools/GIT_Test/GIT_Success.txt [hunk 1]
```diff
@@ -168,3 +168,4 @@ Lee Yong Ler (Retry)
 Chenghao Tan
 RuffaloLavoisier (Jeon sunghwan) was here.
 Giovanni Rasera - Ciao Mondo from Italy - 2022
+Карманов Семён - хуй войне!
```
Comment (hendjoshsr71) [inline Tools/GIT_Test/GIT_Success.txt:171]: While I agree with the sentiment of the comment ([translation](https://translate.google.com/?sl=auto&tl=en&text=%D1%85%D1%83%D0%B9%20%D0%B2%D0%BE%D0%B9%D0%BD%D0%B5&op=translate)), please change it to not include vulgarity/politics.

**Review by hendjoshsr71 — CHANGES_REQUESTED** [review]

File: Tools/GIT_Test/GIT_Success.txt (hunk 1 above)
Comment (cavesLezet) [reply Tools/GIT_Test/GIT_Success.txt]: Ok, I corrected it.

Comment (hendjoshsr71) [reply Tools/GIT_Test/GIT_Success.txt]: Thank you. Appreciated.

**Comment by hendjoshsr71** [comment]
One more fix up. If you could "squash" your commits together to be just one commit that would be great.

[https://ardupilot.org/dev/docs/git-interactive-rebase.html](https://ardupilot.org/dev/docs/git-interactive-rebase.html)

https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together

I personally use Gitkraken for that as I prefer a GUI,
https://www.gitkraken.com/learn/git/git-squash

**Comment by magicrub** [comment]
Also, please only use ASCII text, not UTF.

**Comment by cavesLezet** [comment]
Ok, guys, i fixed all. It wasn't easy to squash commits, but it was a useful experience.

**Review by hendjoshsr71 — APPROVED** [review]

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**